torch/utils/data/datapipes/utils/common.py

```diff
@@ -9,6 +9,7 @@
 from functools import partial
 from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Union
 from torch.utils._import_utils import dill_available
+from torch.testing._internal.common_utils import LazyVal
```

Comment:
Is it kosher to import internal testing utils here? `dill_available` doesn't actually import `dill` so I think it's fine to call it at import time.